client/tailscale/tailscale.go

```diff
@@ -63,6 +65,30 @@ func (c *Client) httpClient() *http.Client {
 	return http.DefaultClient
 }
 
+// BuildURL builds a url to http(s)://<apiserver>/api/v2/... using the given
```

Comment:
These functions allow us to build URLs consistently and make sure that everything gets path escaped.